x-pack/plugins/alerting/server/application/rule/methods/get_schedule_frequency/get_schedule_frequency.ts

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
General comment, it will have been nice to just pass what the function needs instead of getting the full context of rule client in every function.